carma_wm/src/WMListener.cpp

```diff
@@ -111,6 +115,11 @@ void WMListener::setConfigSpeedLimit(double config_lim) const
   worker_->setConfigSpeedLimit(config_lim);
 }
 
+void WMListener::setVehicleParticipationType(std::string participant) const
```

Comment:
Does the WMListener really need a setter for this? If the intent is to load it from the parameter in this class, I don't see why this function is necessary. It could lead to confusion for the user to have a situation where they set the value but it gets overwritten by a parameter anyway.